website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -25,9 +25,27 @@ You must implement the following requirements to create and use cluster peering
 - At least two Kubernetes clusters
 - The installation must be running on Consul on Kubernetes version 0.47.1 or later
 
-### Helm chart configuration
+###  Preparation for install
 
-To establish cluster peering through Kubernetes, deploy clusters with the following Helm values.
+1. After provisioning a Kubernetes cluster and setting up your kubeconfig file to manage access to multiple Kubernetes clusters, export the Kubernetes context names for future use with `kubectl`. 
```

Comment:
```suggestion
1. After provisioning a Kubernetes cluster and setting up your kubeconfig file to manage access to multiple Kubernetes clusters, export the Kubernetes context names for future use with `kubectl`. 
```
Should "Your kubeconfig file should be set up to manage access to multiple Kubernetes clusters" be one of the bullets under the main Prerequisites section? It is a new step that we are assuming the user already did by this point.

Reply:
I think its ok to leave this to the user but I'll add a link to the k8s docs to make this more explicit and reduce confusion.